updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -121,17 +121,20 @@ include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
 See xref:../../../networking/enable-cluster-wide-proxy.adoc#enable-cluster-wide-proxy[Configuring the cluster-wide proxy] to configure the CA to trust the update server.
 ====
 
+// Configuring the Cluster Version Operator (CVO) to retrieve release image signatures
+include::modules/configure-cvo-retrieve-image-signatures.adoc[leveloffset=+1]
+
 [id="next-steps_updating-restricted-network-cluster-osus"]
 == Next steps
 
 Before updating your cluster, confirm that the following conditions are met:
 
 * The Cluster Version Operator (CVO) is configured to use your locally-installed OpenShift Update Service application.
-* The release image signature config map for the new release is applied to your cluster.
+* Your cluster is able to retrieve release image signatures for the update target release images.
```

Comment:
Rather than adding an additional statement about how oc mirror 4.15+ and OSUS 5.0.3+ makes this step unnecessary, I just broadened this condition to state that the cluster can get access to signatures somehow. 

If it's worth specifying the two different ways that can happen depending on your workflow, I can add that in level two bullets underneath this one.